### PR TITLE
[03ebyJQY] Cucumber user testing

### PR DESCRIPTION
**Pull request by sandrely26**
This PR include: Cucumber user tests

There are some scenarios. Most of them are when the voting changes the status.
The user should see 'X' when a voting is 
-open
-voting
-reviewing
-close

Also there are tests for sign in, sign out and sign up

### CODE REVIEW

**Comment by mumoc** [comment]
Just to note, I liked the commits, atomic ones, also you've included the description on the PR, great job !!

**Comment by sandrely26** [comment]
Thank you :)

File: app.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "ratefast",
   "description": "This app allows you to vote for your favorite meals for the whole week",
   "scripts": {
-    "postdeploy": "bundle exec rake db:setup; db:migrate"
+    "postdeploy": "bundle exec rake db:migrate && bundle exec rake db:setup"
```
Comment (mumoc) [inline app.json]: Why this code is in here? your PR is for cucumber test user testing, this config should not be here...

File: features/step_definitions/user_steps.rb [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+  @user = FactoryGirl.create(:user, @visitor)
+end
+
+def create_voting
+  @voting = FactoryGirl.create(:voting)
+  create_items
+end
+
+def change_voting_status_voting
+  @voting = Voting.create
```
Comment (mumoc) [inline features/step_definitions/user_steps.rb]: Check how to create a `FactoryGirl.create(:voting_voting)` which is in voting state...

File: features/step_definitions/user_steps.rb [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+  @voting.vote
+  @voting.save
+  @voting
+  create_items
+end
+
+def change_voting_status_reviewing
+  @voting = Voting.create
```
Comment (mumoc) [inline features/step_definitions/user_steps.rb]: THe same as before here, there are ways to create Factories that represent different states

**Pull request merged**